res/styles/style-index.css

```diff
@@ -1,33 +1,18 @@
+/*Setting the color for h1*/
 h1 {
     color: SeaGreen;
   }
 
+/*Setting the grid for the container class*/  
```

Comment:
Usally I like comments, but the code here is very obvious, so do you think you need the comment? Is it add value?
```suggestion

```

Reply:
hmm, i'll leave it for a while, until i get used to it...
Sometimes i get confused with what i did on the code...
Also only now i realize the importance of comments